ldap_sync/data/cron.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+  <!--
+      Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+  -->
+<odoo>
+
+    <!-- <record id="ldap_sync_cron" model="ir.cron">
```

Comment:
better uncomment and set as inactive, and make it noupdate

Reply:
👍